client/src/lib/util/miscellaneous.ts

```diff
@@ -0,0 +1,8 @@
+export const omit = (
+  obj: { [key: string]: unknown },
+  props: string[]
+): { [key: string]: unknown } => {
+  const temp = { ...obj };
+  props.forEach((prop) => delete temp[prop]);
+  return temp;
+};
```

Comment:
2 rq:

* Sur le fichier : on pourrait plutôt appeler ça `objects.ts`, en s'inspirant de lodash ? On aurait peut-être plus tard `arrays.ts`, etc. C'est tjr difficile de choisir des noms d'utilitaires, mais "miscellaneous" ça ouvre la porte à mettre ici un peu de tout.
* Sur ce helper : on peut en faire une version typée (j'ai pompé sur lodash) :


```suggestion
export const omit = <T extends Record<string, unknown>, K extends keyof T>(
  obj: T,
  props: K[]
): Omit<T, K> => {
  const temp = { ...obj };
  props.forEach((prop) => delete temp[prop]);
  return temp;
};
```